packages/opentelemetry-core/src/common/types.ts

```diff
@@ -16,11 +16,14 @@
 
 import { Exception } from '@opentelemetry/api';
 
+/** @deprecated This enum will be removed prior to v1.0, use the {@link DiagLogLevel}
+ * @see {@link DiagLogLevel} from the api
+ */
 export enum LogLevel {
-  ERROR,
-  WARN,
-  INFO,
-  DEBUG,
+  ERROR = 0,
```

Comment:
Explicitly assigning the inferred values for clarity with the mapping in DiagLogLevel

Reply:
Correct not technically needed, just being more explicit as this always providing the implicit values as explicit avoids future bugs when someone (assuming it's not picked up via code review or tests) inadvertently adds and reorders the names.

And as this enum will be removed as part of the Part 2 followup it's (really) not needed, but I was hoping that it made the rationale and numbering used for DiagLogLevel easier to understand.

Reply:
Removing and making breaking the DiagLogLevel and LogLevel numerical compatibility, due to other comments.